content/1.7/back-office/catalog/stock/stock.md

```diff
@@ -0,0 +1,31 @@
+
+---
+title: Stock
+weight: 1
+---
+
+  
+# SPECS Stock & Stock movement
+
+## Stock
+
+  
+| Stock's type | Definition | When impacted | 
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Physical stock | Number of products that are physically in my stock.| When a merchant can add or remove physical stock 
+| Reserved stock | Number of products that are part of customer orders that aren’t shipped yet.| When client's merchant order products |
+| Available stock | Number of products that are available for sale. Available stock = physical stock - reserved stock | When Physical and Reserved Stock are impacted
+
+
+## Stock movement
+
+**User story**
+
+ As a merchant, I want to associate a stock edition with a movement type to track of the reason of a given physical stock movement.
+
+  Stock movement | When the stock movement is saved | Impact
+|:--------:| :-------------|:-------------|
+Product return | When a user re-stocks a product from an Order that has been flagged as _shipped_ | The quantity re-stocked increases the product's physical stock | 
```

Comment:
```suggestion
Product return | When a user re-stocks a product from an Order that has been flagged as _shipped_ | The quantity re-stocked increases the product's physical (and available) stock | 
```

Reply:
I will do a matrix as suggested, it will be more explicit

Reply:
@jolelievre the impact on available stock should be implicit by definition of "available", but it doesn't hurt to remind the impact here
```suggestion
Product return | When a merchant restocks a product from an Order that has been flagged as _shipped_ | The restocked quantity increases the product's physical stock, and implicitly its available stock | 
```